NitroxLauncher/ServerLogic.cs

```diff
@@ -31,7 +31,7 @@ public void Dispose()
             serverProcess = null;
         }
 
-        internal Process StartServer(bool standalone)
+        internal Process StartServer(bool standalone, string saveDir)
         {
             if (IsServerRunning)
```

Comment:
> Also, it seems that `IsServerRunning` doesn't detect servers that weren't started by the current launcher instance.

`IsServerRunning` is used to determine which page to show in the launcher. It's not meant to prevent multiple server processes.

Reply:
> maybe we should still be able to start servers that don't listen on the same port

I don't think that's the purpose of that class. It's just meant to handle the server being started in and out of the launcher. I'll rename it to `StartServerHandler` to prevent that confusion.